docs/windows.md

```diff
@@ -8,18 +8,24 @@
   - [Antrea Windows demo](#antrea-windows-demo)
 - [Deploying Antrea on Windows Worker Node](#deploying-antrea-on-windows-worker-node)
   - [Prerequisites](#prerequisites)
-  - [Installation as a Service (containerd based runtimes)](#installation-as-a-service-containerd-based-runtimes)
-  - [Installation as a Pod (Docker/containerd based runtimes)](#installation-as-a-pod-dockercontainerd-based-runtimes)
+  - [Installation as a Pod](#installation-as-a-pod)
     - [Download &amp; Configure Antrea for Linux](#download--configure-antrea-for-linux)
     - [Add Windows antrea-agent DaemonSet](#add-windows-antrea-agent-daemonset)
     - [Join Windows worker Nodes](#join-windows-worker-nodes)
```

Comment:
Given that many steps in these sections are commonly required for both pod (docker/containerd) and service deployment methods, is it better to add an environment preparation section in the parent directory?

Reply:
I don't think this is very compatible with my desire to make the recommended installation method (`antrea-windows-containerd-with-ovs.yml`) as straightforward and as easy to follow as possible. With a standard installation, you will typically first create the antrea-agent Windows DaemonSet, then join Windows worker Nodes. With the service installation, everything happens directly at the Windows worker Nodes.

That being said, I realized that the `Installation as a Service` section was missing information, and so I improved it. In particular, I added a link to the `Join Windows worker Nodes` section. PTAL